buildenv/jenkins/openjdk_tests

```diff
@@ -292,23 +292,28 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
                 } else {
                     def gitConfig = scm.getUserRemoteConfigs().get(0)
 
-                    // Adopt windows machines require env here https://github.com/AdoptOpenJDK/openjdk-tests/issues/1803
-                    ref_cache = "${env.HOME}/openjdk_cache"
 
-                    checkout scm: [$class: 'GitSCM',
-                        branches: [[name: "${scm.branches[0].name}"]],
-                        extensions: [
-                            [$class: 'CleanBeforeCheckout'],
-                            [$class: 'CloneOption', reference: ref_cache],
-                            [$class: 'RelativeTargetDirectory', relativeTargetDir: 'openjdk-tests']],
-                        userRemoteConfigs: [[url: "${gitConfig.getUrl()}"]]
-                    ]
-                }
-                jenkinsfile = load "${WORKSPACE}/openjdk-tests/buildenv/jenkins/JenkinsfileBase"
-                if (LABEL.contains('ci.agent.dynamic') && CLOUD_PROVIDER == 'azure') {
-                    //Set dockerimage for azure agent. Fyre has stencil to setup the right environment
-                    docker.image('adoptopenjdk/centos6_build_image').pull() 
-                    docker.image('adoptopenjdk/centos6_build_image').inside {
+                        // Adopt windows machines require env here https://github.com/adoptium/aqa-tests/issues/1803
+                        ref_cache = "${env.HOME}/openjdk_cache"
+
+                        checkout scm: [$class: 'GitSCM',
+                            branches: [[name: "${scm.branches[0].name}"]],
+                            extensions: [
+                                [$class: 'CleanBeforeCheckout'],
+                                [$class: 'CloneOption', reference: ref_cache],
+                                [$class: 'RelativeTargetDirectory', relativeTargetDir: 'openjdk-tests']],
+                            userRemoteConfigs: [[url: "${gitConfig.getUrl()}"]]
+                        ]
+                    }
+                    jenkinsfile = load "${WORKSPACE}/openjdk-tests/buildenv/jenkins/JenkinsfileBase"
+                    if (LABEL.contains('ci.agent.dynamic') && CLOUD_PROVIDER == 'azure') {
+                        //Set dockerimage for azure agent. Fyre has stencil to setup the right environment
+                        docker.image('adoptopenjdk/centos6_build_image').pull()
+                        docker.image('adoptopenjdk/centos6_build_image').inside {
+                            jenkinsfile.testBuild()
+                        }
+                    } else {
+
```

Comment:
This change is not related to the git reference update. Did you change the indentation?

Reply:
No @llxia I didn't change the indentation. 
Any suggestions?

Reply:
This might have occurred while I was trying to resolve the conflicts. Shall I make it the same (to original)?